HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -22,6 +22,7 @@
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/Framework/interface/EventSetupRecord.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/Exception.h"
 
```

Comment:
```suggestion

#include <algorithm>
```
Because of `std::binary_search`.

Reply:
Thanks, done!